build/setup.d/100-install-open-policy-agent.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+curl -L -o opa https://github.com/open-policy-agent/opa/releases/download/v0.16.2/opa_linux_amd64
```

Comment:
Please use `--fail` and the usual boilerplate (`set -euo pipefail`) on top, and add a signature check. You'll also need to `chmod +x` it.

Reply:
added the options